docs/modules/fault-tolerance/partials/nav.adoc

```diff
@@ -1,13 +1,12 @@
 * Configuring for Fault Tolerance
 ** xref:fault-tolerance:backups.adoc[Backups]
-** Persisting Data on a Cluster
+** Persistence
 *** xref:storage:persistence.adoc[Overview]
-*** xref:storage:enabling-persistence.adoc[Quickstart]
-*** xref:storage:configuring-persistence.adoc[Configuration Options]
-*** xref:storage:recover-single-member.adoc[Handling Member Failures]
-*** xref:storage:encryption-at-rest.adoc[Encrypting Data at Rest]
+*** xref:storage:configuring-persistence.adoc[Configuration]
+*** xref:storage:backing-up-persistence.adoc[Backups]
+*** xref:storage:triggering-force-start.adoc[Force-start]
+*** xref:storage:triggering-partial-start.adoc[Partial-start]
 *** xref:storage:copying-persistence.adoc[Copying Data]
-*** xref:storage:backing-up-persistence.adoc[Backing Up Data]
 *** xref:storage:persistence-on-intel.adoc[Intel Optane DC]
 *** xref:storage:performance.adoc[Performance Benchmark]
```

Comment:
Not a right place for the comment but a note - the quality of the images in "persistence performance" chapter is quite poor.